etc/config/c.defaults.properties

```diff
@@ -30,14 +35,16 @@ compiler.cg7.exe=/usr/bin/gcc-7
 compiler.cg7.name=gcc 7.x
 compiler.cg8.exe=/usr/bin/gcc-8
 compiler.cg8.name=gcc 8.x
-compiler.cg9.exe=/usr/bin/gcc-9
+compiler.cg9.exe=/Users/QXZ3F7O/MinGW/bin/mingw32-gcc-9.2.0.exe
 compiler.cg9.name=gcc 9.x
 compiler.cg10.exe=/usr/bin/gcc-10
 compiler.cg10.name=gcc 10.x
-compiler.cg11.exe=/usr/bin/gcc-11
+compiler.cg11.exe=/Users/QXZ3F7O/MinGW/bin/g++.exe
 compiler.cg11.name=gcc 11.x
-compiler.cgdefault.exe=/usr/bin/gcc
+compiler.cgdefault.exe=/Users/QXZ3F7O/MinGW/bin/mingw32-gcc.exe
 compiler.cgdefault.name=gcc default
+compiler.cgtricore.exe=/Users/QXZ3F7O/Desktop/tricore_494_mingw/bin/tricore-elf-gcc.exe
```

Comment:
cross compiler are not defined here. There is a specific config section below with a bunch of gcc-cross